src/grid/rtransform.py

```diff
@@ -32,23 +34,23 @@ class BaseTransform(ABC):
     """Abstract class for transformation."""
 
     @abstractmethod
-    def transform(self, array):
+    def transform(self, r):
         """Abstract method for transformation."""
 
     @abstractmethod
-    def inverse(self, r_array):
+    def inverse(self, r):
```

Comment:
inverse r and transform r are different, so I would prefer them to be different arg name

Reply:
OK, is it good to use `r` for transform/deriv/deriv2/deriv3 methods and `r_array` for inverse?

Reply:
it doesn't matter what name to use. We can use x for [-1, 1] and r for [0, inf]. But mixing these two i think is confusing for people who read the code.

Reply:
I agree it is better not to mix things. I liked `x` better to be honest, but as we are using these 1D grids as a radial grid mostly, I thought `r` would be better.

Reply:
@tovrstra do you have any suggestions?

Reply:
oned(x) need to be transform to generate radial(r). So i think it's ok.
TF:
transform(x) -> r
inverse(r) -> x
INV:
transform(r) -> x
inverse(x) -> r

Reply:
@tczorro see 63bc25c, let me know if anything is left out. Do you know why `flake8` is failing?